.github/ISSUE_TEMPLATE/CONTRIBUTING.md

```diff
@@ -0,0 +1,240 @@
+
+
+# Contribution Guidelines
+
+We welcome contributions from the community! By contributing to this project, you agree to abide by our code of conduct. Please take a moment to review these guidelines to ensure your contributions are well-received.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Code of Conduct](#code-of-conduct)
+- [Using Better Commit](#using-better-commit)
+- [Design Patterns](#design-patterns)
+- [Stack-Specific Guidelines](#stack-specific-guidelines)
+- [Bug Reports](#bug-reports)
+- [Pull Requests](#pull-requests)
+- [How to Contribute](#how-to-contribute)
+
+## Introduction
+
+Thank you for your interest in contributing to the Rasengan.js Templates project! Contributions can be in the form of new templates, bug fixes, documentation improvements, and more.
+
+## Code of Conduct
+
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project, you agree to abide by its terms.
+
+## Using Better Commit
+
+We use the Better Commit CLI to standardize commit messages and branch naming conventions. This ensures that our commit history is clean and easy to follow. Please follow these steps to install and use Better Commit:
+
+### Installation
+
+Install Better Commit globally via npm:
+```bash
+npm install -g better-commits
+```
+
+### Creating a Branch
+
+When creating a new branch, use the structure provided by Better Commit:
+```bash
+better-commit branch
+```
+
+### Making a Commit
+
+Use the Better Commit CLI to make your commits:
+```bash
+better-commit
+```
+
+For more details, refer to the [Better Commit documentation](https://github.com/Everduin94/better-commits).
+
+## Design Patterns
+
+To maintain consistency across our templates, we recommend following these design patterns:
+
+### Container-Presenter Pattern
+
+- **Container Components**: Handle state management and business logic.
+- **Presenter Components**: Handle the rendering of UI elements.
+
+### Atomic Design
+
+- **Atoms**: Basic building blocks (buttons, inputs).
+- **Molecules**: Groups of atoms functioning together (forms, cards).
+- **Organisms**: Groups of molecules forming a distinct section (headers, footers).
+- **Templates**: Page-level layout components.
+- **Pages**: Specific views/screens in the application.
+
+Certainly! Let's structure the Design Patterns section respecting the Atomic Design pattern. Here's how it should look:
+
+---
+
+Recommended project structure for templates following the Atomic Design pattern:
+
+```
+rasenganjs-examples/
+├── template1/
+│   ├── atoms/
+│   │   ├── Button/
+│   │   ├── Input/
+│   ├── molecules/
+│   │   ├── Header/
+│   │   ├── Sidebar/
+│   ├── organisms/
+│   │   ├── HomePage/
+│   │   ├── AboutPage/
+├── template2/
+│   ├── atoms/
+│   │   ├── Icon/
+│   │   ├── Label/
+│   ├── molecules/
+│   │   ├── Navbar/
+│   │   ├── Footer/
+│   ├── organisms/
+│   │   ├── DashboardPage/
+│   │   ├── ProfilePage/
+```
```

Comment:
We have to discuss again for this section.

Reply:
okay